toolchain/stage2/args/linux/BUILD.bazel

```diff
@@ -4,20 +4,7 @@ load("@rules_cc//cc/toolchains/impl:documented_api.bzl", "cc_args_list")
 
 package(default_visibility = ["//toolchain/stage2:__subpackages__"])
 
-# Passing `/dev/null` as sysroot has the side effect of removing all default
```

Comment:
We're sure this doesn't result in any new non-hermetic search paths?

Reply:
I didn't remove it actually. I just reused the one from //toolchain.
With our setup, the only thing that --sysroot=/dev/null does (but still important), is to remove -L/usr/lib that clang auto adds to linker args.

Reply:
Ah ok, sorry I missed it. And yeah I was worried about those auto-add args, thanks for confirming!